source/userguide/kb.rst

```diff
@@ -716,6 +716,45 @@ In the example below, we search for restaurants whose names best match ``firetra
    We can set the ``size`` parameter of the :meth:`execute()` method to specify the maximum number of records.
 
 
+.. _question_answerer_config:
+
+QuestionAnswerer Configurations
+-------------------------------
+
+To override MindMeld's default QuestionAnswerer configuration with custom settings, you can edit the app configuration file by adding a dictionary of custom setting named :data:`QUESTION_ANSWERER_CONFIG`. If no such dictionary is present in the ``config.py`` file, MindMeld loads a QuestionAnswerer with default settings. The following are the default settings:
+
+.. code-block:: python
+
+  QUESTION_ANSWERER_CONFIG = {
+      'model_type': 'elasticsearch',
+      'model_settings': {
+          'query_type': 'keyword'
+      }
+  }
+
+The ``query_type`` can be one of ``keyword`` or ``text``. While specifying the former optimizes your QuestionAnswerer search for keywords or short spans of text, the latter optimizes for searching on larger paragraphs or passages of unstructured text. In order to leverage embeddings-based semantic matching along with surface-level text features matching, you can specify one of the three embedder query types- ``embedder``, ``embedder_text``, or ``embedder_keyword``. For using embeddings of your choice, you can specify embedder configurations in the :data:`QUESTION_ANSWERER_CONFIG` within the key 'model_settings'. For full details about using embedders in QuestionAnswerer, check the :ref:`Leveraging semantic embeddings <semantic_embeddings>` section below. Note that the specified ``query_type`` is also the default ``query_type`` for all QuestionAnswerer calls in your application, but you can always pass in a different query type to your ``qa.get(query_type=...)`` command as desired.
```

Comment:
minor: maybe we should re-phrase the first sentence a little bit to indicate that `keyword` and `text` and embedder types can be specified in `query_type` param? Right now it sounds like only `keyword` and `text` are allowed before we talk about embedder types later in the paragraph.